### ISSUE TITLE
found 0 images

### ISSUE THREAD

**Issue by ghosharkaprava8** [patrickloeber/python-fun#1]
Tried using the code. But for all keywords it says the same " found 0 images"

**Comment by Amit998** [comment]
i think we have to change class name

**Comment by patrickloeber** [comment]
Yes unfortunately my code is outdated. Google changed their html structure in the meantime. So you have to adapt the class names...

**Comment by Amit998** [comment]
Can you help on one thing? 
I resolved the previous issue.
But it was showing only 20 images I want to collect more for training my model .what can I do?

**Comment by 5CHOPP39** [comment]
hi

**Comment by 5CHOPP39** [comment]
i tried using the code when i ran getting the errors
 data = torch.load(FILE)
near that line..what should i do now?
i have doubt what should i include in data.pth
